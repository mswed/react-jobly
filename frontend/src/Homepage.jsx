import { Container } from 'react-bootstrap';
const Homepage = () => {
  return (
    <Container className="d-flex flex-column justify-content-center align-items-center min-vh-100 text-center" style={{ marginTop: '-5rem' }}>
      <h1 className="text-shadow">Jobly</h1>
      <h2 className="text-shadow">All the jobs in one, convenient place.</h2>
    </Container>
  );
};

export default Homepage;
