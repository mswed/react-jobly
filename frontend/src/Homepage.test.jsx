import { it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';
import Homepage from './Homepage';

it('renders without crashing', async () => {
  render(<Homepage />);
});

it('matches snapshot', async () => {
  const { asFragment } = render(<Homepage />);

  expect(asFragment()).toMatchSnapshot();
});

it('displays the correct text', async () => {
  render(<Homepage />);
  expect(screen.getByText('All the jobs in one, convenient place.')).toBeInTheDocument();
});
