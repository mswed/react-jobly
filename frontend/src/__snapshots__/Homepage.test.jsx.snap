// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-column justify-content-center align-items-center min-vh-100 text-center container"
    style="margin-top: -5rem;"
  >
    <h1
      class="text-shadow"
    >
      Jobly
    </h1>
    <h2
      class="text-shadow"
    >
      All the jobs in one, convenient place.
    </h2>
  </div>
</DocumentFragment>
`;
